pkgs/development/python-modules/qasm2image/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, cairocffi
+, cairosvg
+, cffi
+, qiskit
+, svgwrite
+}:
+
+buildPythonPackage rec {
+  pname = "qasm2image";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "nelimeee";
+    repo = "qasm2image";
+    rev = "7f3c3e4d1701b8b284ef0352aa3a47722ebbbcaa";
+    sha256 = "129xlpwp36h2czzw1wcl8df2864zg3if2gaad1v18ah1cf68b0f3";
+  };
+
+  propagatedBuildInputs = [
+    cairocffi
+    cairosvg
+    cffi
+    qiskit
+    svgwrite
+  ];
+
+  # test doesn't pass
+  doCheck = false;
```

Comment:
Why not? Is it something upstream needs to fix? Or should we?

Reply:
The tests fail because Pypi's QISKit package doesn't contain tests modules.